addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientListScreen.js

```diff
@@ -47,8 +47,11 @@ odoo.define('point_of_sale.ClientListScreen', function(require) {
             };
             this.updateClientList = debounce(this.updateClientList, 70);
         }
-
+        
         // Lifecycle hooks
+        async willStart(){
+            await this.env.pos.load_new_partners().catch(r=>console.log("Partners where not updated",r));
```

Comment:
I would agree with this change but I'm not sure if this can be in stable. @pimodoo Do you think it is okay to allow fetching the newly modified/added partners every time the client list is shown?

Reply:
@caburj I am also not sure to put it in stable, because some customers have a lot of partners, and I would not kill the performences on production databases. But in master it is ok for me

Reply:
I have some test databases with 20k partners I can stress test this if you want.

The main issue behind this, and the possible justification for the performance sacrifice, is that syncronization is crucial for POS, an example would be that loyalty points were not synced in the pos (even on the same session), and also wouldn't be synced between different POS if this doesn't load every time (I don't know how much we can trust the caching layer with this for minimizing the performance issue).

Also in databases with many users, there could be several pos sessions changing client names or addresses, and they would be overwriting each other, and not seeing relevant information that may be important.

One example of this is the previously mentioned loyalty points (I have a client on enterprise  with this issue atm)

Does POS loads all partners at once? or does it load with pagination?